android/java/org/chromium/chrome/browser/crypto_wallet/fragments/ApproveTxBottomSheetDialogFragment.java

```diff
@@ -332,11 +333,11 @@ private void fillAssetDependentControls(String asset, View view, int decimals) {
     }
 
     private void rejectTransaction(boolean dismiss) {
-        EthTxService ethTxService = getEthTxService();
-        if (ethTxService == null) {
+        TxService txService = getTxService();
+        if (txService == null) {
             return;
         }
-        ethTxService.rejectTransaction(mTxInfo.id, success -> {
+        txService.rejectTransaction(CoinType.ETH, mTxInfo.id, success -> {
```

Comment:
as I understood we pass `CoinType.ETH` for any EVM compatible network?

Reply:
Yea correct